src/main/scala/cognite/spark/v1/DefaultSource.scala

```diff
@@ -67,6 +67,21 @@ class DefaultSource
   override def createRelation(sqlContext: SQLContext, parameters: Map[String, String]): BaseRelation =
     createRelation(sqlContext, parameters, null) // scalastyle:off null
 
+  private def createSequenceRows(parameters: Map[String, String], config: RelationConfig, sqlContext: SQLContext) = {
+    val sequenceId =
+      parameters.get("sequenceId").map(id => CogniteInternalId(id.toInt))
+        .orElse(
+          parameters.get("sequenceExternalId").map(CogniteExternalId)
```

Comment:
Isn't `sequence` a bit redundant here, if the user has doen `.option("type", "sequencerows").option("sequenceid", "1828")`?

Reply:
Hmm, maybe :thinking: I still find saying just `id` might not be very clear. Should I change it, or?

Reply:
I think so tbh. I think it is just as clear as "sequenceid", and doubt there will be any confusion.

Reply:
Ok, I have changed that (and added some docs)